src/lib/homestore_backend/tests/homeobj_cp_tests.cpp

```diff
@@ -1,5 +1,6 @@
 #include "homeobj_fixture.hpp"
 
+#if 0
```

Comment:
Why this is disabled? is it because same scenario is already covered by other test cases already?

Reply:
Yes we are handling this check as part of other test.